doom.go

```diff
@@ -43006,7 +43034,6 @@ func w_AddFile(filename string) *os.File {
 		lump_p.Fname = fileinfo[i].Fname
 	}
 	lumphash = nil
-	wad_files[wad_file.Fd()] = wad_file
```

Comment:
Isn't this needed to prevent garbage collection?

Reply:
It was originally, but I think the face that we're storing them in `lumpinfo[i].Fwad_file` is sufficient now. I think when I was half-way through doing the clean up, these were still using `uintptr`, so the garbage collector didn't know they were still in use.